src/main/resources/static/_shared.yml

```diff
@@ -4921,3 +4931,8 @@ components:
         - sharedProperty
         - singleOccupancy
         - wheelchairAccessible
+    ApplicationOrigin:
+      type: string
+      enum:
+        - bail
```

Comment:
as per previous comment - perhaps expand these to include hdc, and better differentiate the 2 types of bail e.g. `prisonBail,courtBail,hdc` ?

Reply:
agree - though lets use `homeDetentionCurfew` instead of `hdc` as it's clearer